README.md

```diff
@@ -1,65 +1,58 @@
+## We :heart: Contributors Like You!
 
-## Download
+**We’re eager to work with you**, our user community to improve these materials and develop new ones. Here's how you can help:
 
-We know that many of the users of this repository are just focused on getting the materials and teaching from them.  We've made that easy.
+- **You spotted a mistake:** please feel free to fork the repository and submit a change via Pull Request (not sure how to do that, [we have a course for you](https://services.github.com/on-demand)).
+- **You have an idea to make it better:** we :heart: new ideas! We invite you to open a new [Issue](https://github.com/github/training-kit/issues) to kick off the process.
+- **You just want to help:** check out the [open issues](https://github.com/github/training-kit/issues) for projects you can tackle, review an [open pull request](https://github.com/github/training-kit/pulls), or check out [the project ROADMAP](https://github.com/github/training-kit/projects/1).
 
-1. You can view and teach from the kit, hosted on GitHub, at https://services.github.com/kit/.
-2. You can download the source files [here](https://github.com/github/training-kit/archive/master.zip).
+For more information on contributing to this repository, check out our [CONTRIBUTING guide](https://github.com/github/training-kit/blob/master/CONTRIBUTING.md).
 
-#### Packaging for Viewing Behind Your Firewall
-
-Sometimes you can't download the repository at work because of firewall rules, but you'd like to have a copy of the files that would be able to be served from a web server inside of these firewall rules. To do so, we need to use `script/package`.
-
-1. Run `script/package` to create a release tarball. This will be in the format `release-XXXXXXX.tgz` for you to take wherever you want.
-2. To test this looks okay, create some folders `mkdir -p test_site/kit`.
-3. Untar the release, `tar -xzf release-XXXXXXX.tgz -C test_site/kit`.
-4. Switch into the test_site directory, `cd test_site`.
-5. View the site with `python -m SimpleHTTPServer`. _Note: Some servers are obviously more advanced than others and can handle redirects, smart recognition of `.html` files, etc_
-
-## Contribute
+## :world_map: Finding Your Way
 
-We’re eager to have your help in improving this kit. If you have an idea for a change, start by opening a new [Issue](https://github.com/github/training-kit/issues) so we can discuss and help guide your contribution to the right location. If you have corrections or kit contributions, we'd be glad to receive them via a [Pull Request](https://help.github.com/articles/using-pull-requests). For kit contributions, we ask you to share in our mindset of minimalism.
+This repository contains three primary resources:
 
-### Class Material
+- Our current [on-demand courses](https://services.github.com/on-demand/) can be found in the [paths directory](https://github.com/github/training-kit/tree/master/paths)
```

Comment:
@brntbeer so in my brain I was thinking that we should link back to the parent repo just in case someone is viewing the README in their fork??? Now that I say it out loud, it is a dumb idea 😜  Thank you for catching that!